Mathlib/Combinatorics/SimpleGraph/Connectivity/Connected.lean

```diff
@@ -717,11 +712,17 @@ section BridgeEdges
 /-- An edge of a graph is a *bridge* if, after removing it, its incident vertices
 are no longer reachable from one another. -/
 def IsBridge (G : SimpleGraph V) (e : Sym2 V) : Prop :=
-  e ∈ G.edgeSet ∧
-    Sym2.lift ⟨fun v w => ¬(G \ fromEdgeSet {e}).Reachable v w, by simp [reachable_comm]⟩ e
+  Sym2.lift ⟨fun v w => ¬ (G.deleteEdges {e}).Reachable v w, by simp [reachable_comm]⟩ e
 
 theorem isBridge_iff {u v : V} :
-    G.IsBridge s(u, v) ↔ G.Adj u v ∧ ¬(G \ fromEdgeSet {s(u, v)}).Reachable u v := Iff.rfl
+    G.IsBridge s(u, v) ↔ ¬(G \ fromEdgeSet {s(u, v)}).Reachable u v := Iff.rfl
+
+theorem not_edge_isBridge (e : Sym2 V) (hne : ¬(e ∈ G.edgeSet)) : G.IsBridge e := by
```

Comment:
```suggestion
theorem isBridge_of_not_mem_edgeSet {e : Sym2 V} (he : e \nin G.edgeSet) : G.IsBridge e := by
```
but I don't think this result is true. This will depend on whether there is another path between the endpoints of `z`